src/renderer/store/modules/Note.js

```diff
@@ -3,11 +3,13 @@ import converter from '@/converter';
 import path from 'path';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { remote } from 'electron';
+import packageJson from '../../../../package';
 
-const octokit = require('@octokit/rest')({
-  requestMedia: 'application/vnd.github.v3+json',
-  headers: {
-    'user-agent': 'octokit/rest.js v1.2.3',
+const Octokit = require('@octokit/rest');
+let octokit = new Octokit({
```

Comment:
I think you can remove the `new Ocktokit` and do this into the if/else statement below.

Reply:
Just declare a simple object

Reply:
Not sure I follow. So, like this?:

```
let octokit = Octokit({
  userAgent: 'code-notes'.concat(packageJson.version),
  mediaType: {
    format: 'application/vnd.github.v3+json',
  },
});
```

Reply:
leave userAgent, and mediaType there correct? to avoid repetition?

Reply:
I'm struggling with this suggestion do you want to put a snippet how you want it on jsfiddle or here?

Reply:
Maybe I got it commit coming thru : )